tools/generators/release-package/index.ts

```diff
@@ -0,0 +1,328 @@
+import {
+  Tree,
+  formatFiles,
+  names,
+  updateJson,
+  ProjectConfiguration,
+  joinPathFragments,
+  visitNotIgnoredFiles,
+  createProjectGraphAsync,
+  reverse,
+  installPackagesTask,
+  readJson,
+  stripIndents,
+} from '@nrwl/devkit';
+
+import * as tsquery from '@phenomnomnominal/tsquery';
+
+import { getProjectConfig, workspacePaths } from '../../utils';
+
+import { PackageJson, TsConfig } from '../../types';
+
+import tsConfigBaseAll from '../tsconfig-base-all';
+
+import { ReleasePackageGeneratorSchema } from './schema';
+import { execSync } from 'child_process';
+
+interface NormalizedSchema extends ReturnType<typeof normalizeOptions> {}
+
+export default async function (tree: Tree, schema: ReleasePackageGeneratorSchema) {
+  const options = normalizeOptions(tree, schema);
+
+  assertProject(tree, options);
+
+  const tasks: Array<(tree: Tree) => void> = [];
+
+  if (options.phase === 'initial') {
```

Comment:
Is there a reason not to name the first phase "preview"?

Reply:
its initial release 

part of preview development phase